packages/api-runner/lib/constants/contracts.js

```diff
@@ -1,66 +1,129 @@
-import { contractAddresses } from '@pooltogether/current-pool-data'
+import { contractAddresses } from "@pooltogether/current-pool-data";
```

Comment:
Looks like the `.prettierrc` didn't make it into git or something. We can copy over the one from the `pool-app` and add it to git here.